.github/workflows/deploy.yml

```diff
@@ -30,15 +30,14 @@ jobs:
     steps:
       - name: Checkout git repository
         uses: actions/checkout@v4
-      - name: Install Ruby
-        uses: ruby/setup-ruby@v1
-        with:
-          bundler-cache: true
-      - name: Install Node.js
-        uses: actions/setup-node@v4
-        with:
-          node-version-file: ".nvmrc"
-          cache: "npm"
+      - name: Install Ruby and nodejs
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y npm ruby ruby-dev
```

Comment:
Why do we need to use `apt-get install` instead of `setup-*` actions?
We can update Ruby version by https://github.com/apache/arrow-site/blob/main/.ruby-version and Node.js version by https://github.com/apache/arrow-site/blob/main/.nvmrc .